friendship-quiz.js

```diff
@@ -1,25 +1,28 @@
-// questions to be randomly asked on page load
-var questionsList = ["What are your/your friend's favorite foods?", "What are your/your friend's favorite animals?"];
 
+var questionsList = ["What are your/your friend's favorite foods?", "What are your/your friend's favorite animals?"];
 
-//randomly chooses a question to display
 function init() {
   var randomIndex = Math.floor(Math.random() * questionsList.length);
   document.getElementById("question").innerHTML = questionsList[randomIndex];
 }
 
 window.onload = init;
 
-var answerList = []; //initialize answerList to be empty
-
-/* ADD YOUR CODE BELOW */
-
-//adds user-submitted answer to answerList
+var answerList = [];
 function submitAnswer() {
-
+  var a = document.getElementById('answerInput').value;
+  console.log(a);
+  answerList.push(a);
+  console.log(answerList);
 }
 
-//checks if user-submitted guess is in answerList
 function checkGuess() {
-
+  var x = document.getElementById('guessInput').value;
+  console.log(x);
+  for (var i = 0; i < answerList.length; i++){
+    if (x == answerList[i]){
+      alert("Correct! Wow, you really must know your friend :O");
+  // console.log(""); = alert("");
```

Comment:
You have a line that (I think) is a note to yourself, which you should remove for the final product.